templates/website/about-privacy.html

```diff
@@ -113,6 +113,10 @@
 
 <p><a href="http://www.mysociety.org/privacy-online/">More general information on how third party services work</a></p>
 
+<h2>Email Opt-Out</h2>
+
+<p>As part of the process of sending your message to your representative we will send you a limited number of emails <a href="">as described here</a>. If during this process you wish to stop receiving emails from us, or you are a representative who no longer wishes to receive email via WriteToThem, please contact us at <a href="mailto:support@writetothem.com">support@writetothem.com</a>.</p>
```

Comment:
This is missing a `href`?
Could the address use the `OPTION_CONTACT_EMAIL` address?